angularjs/angular-tests.ts

```diff
@@ -287,6 +287,35 @@ var scope: ng.IScope = element.scope();
 var isolateScope: ng.IScope = element.isolateScope();
 
 
+// $timeout signature tests
+module TestTimeout {
+    var result: any;
+
+    interface TResult {
+        a: number;
+        b: string;
+        c: boolean;
+    }
+    var fnTResult: (...args: any[]) => TResult;
+    var promiseAny: angular.IPromise<any>;
+    var $timeout: angular.ITimeoutService;
+
+    // $timeout
+    result = <angular.IPromise<any>>$timeout();
```

Comment:
@vvakame I believed that `result = $timeout()` only checks the possibility to call $timeout as function, but does not verify that the return type as expected, isn't it?